src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/star.svg');
+  background-size: cover;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('../images/star.svg');
```

Comment:
The rule for `.stars--0 .stars__star:nth-child(-n + 0)` is unnecessary since it applies the same background image as the default for `.stars__star`. You can safely remove this rule to clean up your CSS.